src/System.CommandLine/Help/HelpBuilder.cs

```diff
@@ -578,7 +583,7 @@ private string GetFirstColumnText(IIdentifierSymbol symbol)
         /// </summary>
         /// <param name="symbol">The symbol to get the description for.</param>
         /// <param name="context">A parse result providing context for help formatting.</param>
-        private string GetSecondColumnText(IIdentifierSymbol symbol, HelpContext context)
+        public string GetIdentifierSymbolDescription(IIdentifierSymbol symbol, HelpContext context)
```

Comment:
This will still call into customizations via `GetArgumentDescription` which calls `GetArgumentDefaultValueText`.

If these are intended to be default implementations, the cleanest approach would be to make them `static`. The `LocalizationResources` would need to be passed in but not other state (e.g. `_customizationsBySymbol`) specific to the current `HelpBuilder`

Reply:
Related to that, like `DefaultLayout`, they should probably include the word "default" in the method names.

Reply:
I renamed methods and made them static, as agreed yesterday. Required a bit of reshuffling though.